src/psyclone/tests/domain/lfric/transformations/lfric_extract_test.py

```diff
@@ -57,6 +55,18 @@
 # API names
 DYNAMO_API = "dynamo0.3"
 
+
+@pytest.fixture(scope="function", autouse=True)
+def clear_region_name_cache():
```

Comment:
How come this isn't needed in lfric_extract_driver_creator_test.py?

Reply:
Because none of the tests checks for the region names. The region names are the same as the one used in extraction, and they are  just passed in and used unmodified.